src/latex_writer.rs

```diff
@@ -0,0 +1,194 @@
+use crate::{
+    AtomicExpression, BinaryExpression, BinaryOperator, Error, Expression, ExpressionKind,
+    ExpressionWriter, ParenExpression, Tableau, TableauWriter, TruthValueExpression,
+    UnaryExpression, UnaryOperator,
+};
+
+#[derive(Clone, Debug, Default)]
+pub struct LatexExpressionWriter {
+    buffer: String,
+}
+
+impl LatexExpressionWriter {
+    pub fn write_truth_value(&mut self, truth_value: &TruthValueExpression) {
+        if truth_value.value == true {
+            self.buffer += "T";
```

Comment:
In the future we need a system that maps different characters to certain operations and results.